src/lib/components/Text/Text.js

```diff
@@ -7,12 +7,56 @@ import styles from './Text.module.scss';
 
 import { getColor } from '../Treebank/config';
 
-// eslint-disable-next-line react/prop-types
-const wordToSpan = (config, active, toggleActive, word) => {
-  const { $: { id, form, postag } } = word;
+const formatId = (id) => id.padStart(4, '0');
+
+const formatInsertionId = (insertionId) => {
+  if (/-/.test(insertionId)) {
```

Comment:
The reasoning behind this if statement comes from [this function](https://github.com/alpheios-project/arethusa/blob/1bfbc6685bfc7068c9befc36ed226dc162c24515/app/js/arethusa/factories/treebank_retriever.js#L174) in Arethusa. The comment explains that sometimes the `insertion_id` can be in be preceded by the sentence ID and a `-`.

Reply:
ok, I think it's probably unlikely that we'll encounter this anymore but I guess there isn't harm in having the check.

Reply:
I did a quick grep through `gorman-trees` and the AGLDT and I didn't find any. If you don't think it's likely we'll encounter any, maybe it would be better to remove this line for simplicity?

I can imagine a situation where some future treebank implementor looks at this code and copies it, someone copies the future code, etc. and it leads to a zombie if statement in all future implementations.

Reply:
I think it would be fine to remove it. The alternate syntax was from an earlier iteration of the Arethusa code for this feature.